packages/react-composites/src/composites/localization/locales/en-US/strings.json

```diff
@@ -37,7 +37,10 @@
     "removedFromCallTitle": "You were removed",
     "soundLabel": "Sound",
     "startCallButtonLabel": "Start call",
-    "microphoneToggleInLobbyNotAllowed": "Cannot mute or unmute while in lobby."
+    "microphoneToggleInLobbyNotAllowed": "Cannot mute or unmute while in lobby.",
```

Comment:
Ideally, this change should not be a part of this PR.
The purpose of this PR is to convert people pane into a common component.
Doing so would have kept this change as a `none` change (doesn't affect the public API in any way)

The strings can then be introduced in the PR that actually adds this people pane to call composite.

Reply:
And those strings would be conditionally compiled.

Reply:
That's a good point! Thanks Anjul. Will remove these strings and only focus on moving the PeoplePaneContent out as a common with no Api change for this PR.